tests/CMakeLists.txt

```diff
@@ -92,6 +92,7 @@ if (UNIX
     add_subdirectory(enclaveparam)
     add_subdirectory(file)
     add_subdirectory(getenclave)
+    add_subdirectory(helper)
```

Comment:
Can you rename the test to be more specific? "helper" is not intuitive to me what the test intends to do.

Reply:
@mingweishih I thought serialization/deserialization to be helper methods hence the name, I can think of  utils, common ( the files added are common/custom_claims.c ) or custom_claims. Any suggestions ?

Reply:
custom_claims is ok with me

Reply:
renamed to custom_claims